Content.Shared/CCVar/CCVars.cs

```diff
@@ -2226,6 +2226,14 @@ public static readonly CVarDef<float>
         public static readonly CVarDef<string> ReplayAutoRecordTempDir =
             CVarDef.Create("replay.auto_record_temp_dir", "", CVar.SERVERONLY);
 
+
+        /// <summary>
+        ///     The amount of time between NPC Silicons draining their battery in seconds.
```

Comment:
What does this mean?

Reply:
Death accommodated for IPCs being NPCs. NPCs get simplified charge drain using this as a delay between updates. This also skips heat calculations on batteries

Reply:
I completely missed the "NPC" part when reading , so this just sets an update time?

What's the battery heat calculation?